src/views/Pipeline.vue

```diff
@@ -569,11 +575,13 @@ export default defineComponent({
     // setting the current job as empty because when coming back to the pipeline page the currentJob
     // state does not gets updated and hence the job configuration component takes it space in DOM
     await this.store.dispatch('job/updateCurrentJob', { job: {} });
+    emitter.on("productStoreChanged", this.refreshJobs);
```

Comment:
If we are using refreshJobs, we should handle isRetrying properly. This flag will show spinner on retry button when product store changed, even though user hasn't clicked the button.

Reply:
OK, Sir, I will correct it